deploy/tiller-crd.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           value: kube-system
         - name: TILLER_HISTORY_MAX
           value: "0"
-        image: gcr.io/kubernetes-helm/tiller:v2.7.2
```

Comment:
There are differences in the jsonnet that don't appear in this yaml file - can you run `make` in this directory? (perhaps remove `tiller-crd.yaml` first to _force_ it to be regenerated)